lib/routes-deprecated/pcr/news-cn.js

```diff
@@ -1,28 +0,0 @@
-const got = require('@/utils/got');
```

Comment:
Please also remove the entries in
https://github.com/DIYgod/RSSHub/blob/a5219f465f057065f7f7c9c068586bf5e2b0a520/lib/router.js#L997-L998